Prüfschritte/de/11.7 Benutzerdefinierte Einstellungen.adoc

```diff
@@ -2,17 +2,9 @@
 include::include/author.adoc[]
 include::include/attributes.adoc[]
 
-== Grundlage des Prüfschritts
-
-Grundlage des Prüfschritts ist die Tabelle A.2 im Annex A der EN 301 549
-V3.1.1.
-In dieser Tabelle wird auf Abschnitt
-11.7 "User preferences" verwiesen.
-
 == Was wird geprüft?
 
-Die App soll Systemweite Einstellungen berücksichtigen.
-Im Einzelnen sind dies:
+Die App soll Systemweite Einstellungen berücksichtigen. Im Einzelnen sind dies:
```

Comment:
gemeint ist wohl: systemseitige Einstellungen.
Ich würde schreiben: Die App soll systemseitge Einstellungen, z.B. in den Bedienungshilfen, berücksichtigen. Im Einzelnen können dies sein:
[ dann Liste gemäß EN; nicht alles wird immer einstellbar sein und neues hinzukommen ,also allgemeiner fassen über "können dies sein" und ruhig den Text der EN widerspiegeln]
soweit diese nicht von Einstellungen durch Nutzende überschrieben werden.